### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by snxx-lppxx**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

## Problem/Feature

Problems could be due to a bug in .gitignore

## Description of Changes: 

- .gitignore: `vsvode` -> `vscode`
- changed variable declaration
- changed the concatenation

## Checklist

- [x] My code follows the [Contribution Guide](https://github.com/twitchdev/twitch-cli/blob/master/CONTRIBUTING.md#Requirements)
- [ ] I have self-reviewed the changes being requested
- [ ] I have made comments on pieces of code that may be difficult to understand for other editors
- [ ] I have updated the documentation (if applicable)

### CODE REVIEW

**Comment by lleadbet** [comment]
Hi @snxx-lppxx! Appreciate the PR. What's the purpose of these changes? Also- can you please fill out the PR template for me?

**Comment by snxx-lppxx** [comment]
Hello @lleadbet, I changed:
1. fixed a typo in .gitignore
2. changed variable declaration
3. changed the concatenation

**Comment by lleadbet** [comment]
Can you please update the PR template?

**Comment by snxx-lppxx** [comment]
@lleadbet, changed. Sorry for the inconvenience.

File: internal/events/types/subscribe/sub_event.go [hunk 1]
```diff
@@ -34,10 +34,12 @@ var triggerMapping = map[string]map[string]string{
...
-	var event []byte
-	var err error
-	var giftUserID string
-	var giftUserName string
+    var (
+        event        []byte
+        err          error
+        giftUserID   string
+        giftUserName string
+    )
```
Comment (andrew000) [inline internal/events/types/subscribe/sub_event.go:42]: The author of the repository has his own style of coding and does not need to change it

File: cmd/token.go [hunk 2]
```diff
@@ -9,10 +9,12 @@ import (
...
-var isUserToken bool
-var userScopes string
-var revokeToken string
-var overrideClientId string
+var (
+    isUserToken      bool
+    userScopes       string
+    revokeToken      string
+    overrideClientId string
+)
```
Comment (andrew000) [inline cmd/token.go:17]: The author of the repository has his own style of coding and does not need to change it

File: internal/request/request.go [hunk 3]
```diff
@@ -16,7 +16,7 @@ func NewRequest(method string, url string, body io.Reader) (*http.Request, error
 	}
 	version := util.GetVersion()
 
-	req.Header.Set("User-Agent", "twitch-cli/"+version)
+	req.Header.Set("User-Agent", "twitch-cli/" + version)
```
Comment (andrew000) [inline internal/request/request.go:19]: Not a TYPO or Format error in Go

**Review by andrew000 — COMMENTED** [review]
Unnecessary and uncritical changes.

**Comment by snxx-lppxx** [comment]
Hi @andrew000, some of the changes are refactoring. to make it easier to read.

**Comment by lleadbet** [comment]
Closing this as it doesn't provide much in terms of changes; will fix the typo in another PR.

**Pull request closed without merging**